File: code/1-50/02.cpp
#include <bits/stdc++.h>

int main() {
	int a = 1, b = 1, sum = 0;
	while (a <= 4E6) {
		int tmp = a;
		a += b;
		b = tmp;
		if (a % 2 == 0) sum += a;
	}
	printf("%d", sum);
	return 0;
}

File: code/1-50/02.py
if __name__ == '__main__':
	f = [1, 2]
	while True:
		f.append(f[-1] + f[-2])
		if f[-1] > 4 * 10 ** 6: break
	print(sum(i for i in f if i % 2 == 0))